Cargo.toml

```diff
@@ -50,6 +50,7 @@ tabwriter = "1"
 fastrand = "1.4.1"
 crossbeam-channel = "0.5"
 seccomp = { version = "0.1.0", path = "./seccomp" }
+ipc-channel = "0.15.0"
```

Comment:
I'd like to make `ipc-channel` depend on only `dev-dependencies`, is it possible?

Reply:
Yes it can be done. Take a look at the latest commit. I may want to use ipc-channel in other part of Youki's code, but let's keep it as dev-dependencies for now, and discuss this when we get there.